README.md

```diff
@@ -398,6 +398,12 @@ Check the [constraint rule comparison matrix](rule_comparison.md) for language-s
 
   // x must be a valid UUID (via RFC 4122)
   string x = 1 [(validate.rules).string.uuid = true];
+  
+  // x must be a valid HTTP header name (via RFC 7230)
+  string x = 1 [(validate.rules).string.http_header_name = true];
```

Comment:
If you don't mind bike shedding a little, one suggestion I would have is we don't make this `http_header_name`, but instead make it a string value and call it `well_known_regex`. This way, we only implement it one time for every language, and every language has a map from this name to patterns, so in future we can grow this with a few line of changes.